strapi/strapi
# feat(content-releases): add functionality to publish release button

### What does it do?

Link publish button in release details view with its endpoint.

### How to test it?

You should have an EE license and the feature enabled by setting contentReleases as enabled on the feature file.
```
// myproject/config/features.js
module.exports = ({ env }) => ({
  future: {
    contentReleases: true
  },
});
```

1. Create a new release and add entries to it
2. Publish this release clicking on the "Release" button on the top right
3. You should see a success message  and now the release is on "only-view" mode
4. All the entries related should be published/unpublished
